SourceX/display.cpp

```diff
@@ -206,7 +227,9 @@ bool SpawnWindow(const char *lpWindowName)
 		Uint32 rendererFlags = SDL_RENDERER_ACCELERATED;
 
 		vsyncEnabled = 1;
+#ifndef __vita__
 		DvlIntSetting("vsync", &vsyncEnabled);

```

Comment:
Perhaps add `DEFAULT_VSYNC_ENABLED` instead?

Reply:
I've removed vita guard. That's a leftover from my experiments and should'n affect anything.